spec/fixtures/vcr/track_events.yml

```diff
@@ -2,11 +2,11 @@
 http_interactions:
 - request:
     method: post
-    uri: https://www.leanplum.com/api?action=multi&apiVersion=1.0.6&appId=<LEANPLUM_APP_ID>&clientKey=<LEANPLUM_PRODUCTION_KEY>&devMode=false&time=1502510400
+    uri: https://www.leanplum.com/api?action=multi&apiVersion=1.0.6&appId=<LEANPLUM_APP_ID>&clientKey=<LEANPLUM_PRODUCTION_KEY>&devMode=false&time=1502521200
     body:
       encoding: UTF-8
-      string: '{"data":[{"action":"track","event":"purchase","userId":123456,"time":"1502510400","params":{"some_timestamp":"2015-05-01
-        01:02:03"}},{"action":"track","event":"purchase_page_view","userId":54321,"time":"1502509800"}]}'
+      string: '{"data":[{"action":"track","event":"purchase","userId":123456,"time":"1502521200","params":{"some_timestamp":"2015-05-01
```

Comment:
this should not change - right now if i run these specs it will fail, because of my time zone, which sucks.  i used `Timecop` to freeze the spec run time, but maybe it needs an option about which time zone to use or something... either way this change should not be necessary.

Reply:
https://github.com/travisjeffery/timecop/issues/41
https://github.com/travisjeffery/timecop/issues/182